test/Interop/Cxx/foreign-reference/inheritance.swift

```diff
@@ -23,7 +23,9 @@ InheritanceTestSuite.test("Templated cast to base") {
   let sc: BaseT = cast(s)
   expectFalse(sc.isBase)
   let sx: BaseT = cxxCast(s)  // should instantiate I to SubT and O to BaseT
-  expectFalse(sc.isBase)
```

Comment:
Was this a typo from a previous patch?

Reply:
Yeah, I believe this was a typo previously, `sx` was unused.